src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -37,13 +39,26 @@ def get(self, request, project, version):
         except Release.DoesNotExist:
             raise ResourceDoesNotExist
 
+        ids_from_GroupResolution = GroupResolution.objects.filter(
+            release=release,
+        ).values_list(
+            'group_id', flat=True
+        )
+
+        ids_from_GroupCommitResolution = GroupCommitResolution.objects.filter(
+            commit_id__in=ReleaseCommit.objects.filter(
```

Comment:
If we make this query somewhere else, mind pointing it out for me? Becuase it's bad and we should fix it. :)

The correct answer is to add an index to this column if we need to. Especially if you're trying to say "give me the groups for this commit id". We just don't currently have an index for this.